_includes/header.html

```diff
@@ -11,7 +11,7 @@
         <a class="page-link" href="https://github.com/git-lfs/git-lfs/tree/main/docs?utm_source=gitlfs_site&amp;utm_medium=docs_link&amp;utm_campaign=gitlfs">Docs</a>
         <a class="page-link" href="https://github.com/git-lfs/git-lfs/discussions?utm_source=gitlfs_site&amp;utm_medium=discussions_link&amp;utm_campaign=gitlfs">Discussions</a>
         <a class="page-link" href="https://github.com/git-lfs/git-lfs/wiki?utm_source=gitlfs_site&amp;utm_medium=wiki_link&amp;utm_campaign=gitlfs">Wiki</a>
-        <a class="page-link" href="https://github.com/git-lfs/git-lfs/releases/latest?utm_source=gitlfs_site&amp;utm_medium=downloads_link&amp;utm_campaign=gitlfs">Downloads</a>
+        <a class="page-link" href="https://github.com/git-lfs/git-lfs?utm_source=gitlfs_site&amp;utm_medium=installation_link&amp;utm_campaign=gitlfs#installing">Installation</a>
```

Comment:
Rather than removing the "Downloads" link entirely, I think we should change this to a "Releases" link.
```suggestion
        <a class="page-link" href="https://github.com/git-lfs/git-lfs?utm_source=gitlfs_site&amp;utm_medium=installation_link&amp;utm_campaign=gitlfs#installing">Installation</a>
        <a class="page-link" href="https://github.com/git-lfs/git-lfs/releases?utm_source=gitlfs_site&amp;utm_medium=releases_link&amp;utm_campaign=gitlfs">Releases</a>
```

That renders like this:

<img width="1137" alt="Screen Shot 2023-02-04 at 1 23 25 PM" src="https://user-images.githubusercontent.com/28857117/216790128-24613c92-8069-4127-92d7-6b3311433f09.png">